releases/0.9.0-incubating/INSTALL.md

```diff
@@ -64,6 +64,20 @@ Download the [signature](https://dist.apache.org/repos/dist/dev/incubator/openwh
 gpg --verify openwhisk-0.9.0-incubating-sources.tar.gz.asc openwhisk-0.9.0-incubating-sources.tar.gz
 ```
 
+You should receive the output messages similar to the following:
+
+```
+﻿gpg: Signature made <time to generate the key> using RSA key ID <key ID>
+gpg: Good signature from "Release manager's name (Release manager of OpenWhisk) <Release manager's Email address>"
+```
+
+It is acceptable that gpg command pops-up some additional warning messages:
```

Comment:
Why is acceptable? I'd take the point of the view the key should be verifiable (that is the release manager's key should be available and valid in a gpg directory).